repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/RTScanner.java

```diff
@@ -33,7 +33,7 @@
  * This class is the backbone for the realtime scanner.
  */
 public class RTScanner {
-
+    public static Boolean kubernetesmode = false;
```

Comment:
never have static mutable fields (aka never share global state). A refactoring is needed to have a non-static field, maybe in another class (such as a Config class)

Reply:
Hi :) , I simply added a setter method to those related classes for setting the kubernetes mode, then we can have the all those field as private, since we need to pass it over somehow if we don't want it global.